packages/suite-web/e2e/tests/settings/t1b1-device-settings.test.ts

```diff
@@ -35,14 +50,15 @@ describe('T1B1 - Device settings', () => {
         cy.getTestElement('@suite/modal/confirm-action-on-device');
         cy.task('pressYes');
     });
+
     /*
      * 1. navigate to settings/device screen and wait for it to load
      * 2. Select & click Choose from gallery in Customization section
      * 3. Select Doge homescreen
      * 4. Confirm on device
      * 5. Wait for success notification Settings applied
      */
-    it('change homescreen', () => {
+    it.skip('change homescreen', () => {
```

Comment:
this never worked and was effectively skipped by wrong group on the first line

Reply:
just out of curiosity, I was looking into why it doesn't work... 
it's throwing an error from dataUrlToImage method, where "blob:http://localhost:8000/dfb5ddad-975f-4dd9-8db2-9f6ac4c8b278" is used as image src, but somehow URL.createObjectURL() is not working the same in Cypress, so on that URL, the image is not available